corehq/apps/app_manager/templates/app_manager/partials/mobile_report_configs.html

```diff
@@ -1,66 +1,84 @@
 {% load hq_shared_tags %}
 {% load i18n %}
 
-<div class="form-horizontal">
-    <div class="save-button-holder clearfix">
-        <div data-bind="saveButton: saveButton" id="save-button"></div>
-    </div>
-    <fieldset>
-        <div class="control-group" id="module-name">
-            <label class="control-label">{% trans "Module Name" %}</label>
-            <div class="controls">
-                <input type="text" data-bind="value: currentModuleName">
-            </div>
+<ul class="nav nav-tabs">
+    <li class="active"><a href="#settings" data-toggle="tab">{% trans "Settings" %}</a></li>
+    <li><a href="#reports" data-toggle="tab">{% trans "Reports" %}</a></li>
+</ul>
+
+<div class="save-button-holder clearfix">
+    <div data-bind="saveButton: saveButton" id="save-button"></div>
+</div>
+
+<div class="tab-content">
+    <div class="tab-pane active" id="settings">
+        <div class="form-horizontal">
+            <fieldset>
+                <div class="control-group" id="module-name">
+                    <label class="control-label">{% trans "Module Name" %}</label>
+                    <div class="controls">
+                        <input type="text" data-bind="value: currentModuleName">
+                    </div>
+                </div>
+                {% if request|feature_preview_enabled:"MODULE_FILTER" %}
+                <div class="control-group">
+                    <label class="control-label">{% trans "Module Filter" %}</label>
+                    <div class="controls">
+                        <input type="text" data-bind="value: currentModuleFilter">
+                    </div>
+                </div>
+                {% endif %}
+                {% include 'app_manager/partials/nav_menu_media.html' with ICON_LABEL="Icon" AUDIO_LABEL="Audio" %}
+            </fieldset>
         </div>
-        {% if request|feature_preview_enabled:"MODULE_FILTER" %}
-        <div class="control-group">
-            <label class="control-label">{% trans "Module Filter" %}</label>
-            <div class="controls">
-                <input type="text" data-bind="value: currentModuleFilter">
-            </div>
+    </div>
+
+    <div class="tab-pane" id="reports">
+        <div class="form-horizontal">
+            <fieldset>
+                <table class="table" id="report-list">
```

Comment:
Here's the reports table using ```table-condensed```:

<img width="1049" alt="screen shot 2015-11-23 at 1 42 35 pm" src="https://cloud.githubusercontent.com/assets/1757035/11345995/5c45e0bc-91e8-11e5-8b8f-58be4a3345ab.png">

I'm not a big fan because I have trouble seeing where one report config ends and another begins; they all flow together for me.